storage/dkg_keys.go

```diff
@@ -5,6 +5,16 @@ import (
 )
 
 type DKGKeys interface {
+	// insert the DKG private key to database when DKG was completed, and a DKG private key was successfully generated.
 	InsertMyDKGPrivateInfo(epochCounter uint64, key *dkg.DKGParticipantPriv) error
-	RetrieveMyDKGPrivateInfo(epochCounter uint64) (*dkg.DKGParticipantPriv, error)
+
+	// insert a record in database to indicate that when DKG was completed, the node failed DKG, there is no DKG key generated.
```

Comment:
I didn't get the meaning of the description, is it possible to rephrase it?
```suggestion
	// insert a record in database to indicate that when DKG was completed, the node failed DKG, there is no DKG key generated.
```